iot/contiki/README.md

```diff
@@ -0,0 +1,162 @@
+The Contiki Operating System for IoT
+====================================
+
+Contiki is an open source operating system that runs on tiny low-power
+microcontrollers and makes it possible to develop applications that
+make efficient use of the hardware while providing standardized
+low-power wireless communication for a range of hardware platforms.
+
+Contiki is used in numerous commercial and non-commercial systems,
+such as city sound monitoring, street lights, networked electrical
+power meters, industrial monitoring, radiation monitoring,
+construction site monitoring, alarm systems, remote house monitoring,
+and so on.
+
+For more information, see the Contiki website:
+
+[http://contiki-os.org](http://contiki-os.org)
```

Comment:
I think you can replace: `[http://contiki-os.org](http://contiki-os.org)` with: `http://contiki-os.org`

Reply:
Think is better to add a hyperlink so when they follow the tutorial they can directly click and go to the Contiki website :+1: